src/common/components/cards/result-section.tsx

```diff
@@ -4,8 +4,6 @@ import { css } from '@uifabric/utilities';
 import { NamedFC } from 'common/react/named-fc';
 import * as React from 'react';
 
-import { TargetAppData } from '../../../common/types/store-data/unified-data-interface';
```

Comment:
Edits in this file seems unrelated to the rest of the change. Opportunistic cleanup or is it actually needed?

Reply:
The former; it was a bit confusing why these properties were declared twice, effectively speaking.